nucypher/utilities/logging.py

```diff
@@ -163,15 +176,43 @@ def _ensure_dir_exists(path):
     pathlib.Path(path).mkdir(parents=True, exist_ok=True)
 
 
-def get_json_file_observer(name="nucypher.json", path=USER_LOG_DIR):
+def get_json_file_observer(name=DEFAULT_JSON_LOG_FILENAME, path=USER_LOG_DIR):
     _ensure_dir_exists(path)
     logfile = LogFile(name=name, directory=path, rotateLength=MAXIMUM_LOG_SIZE, maxRotatedFiles=MAX_LOG_FILES)
     observer = jsonFileLogObserver(outFile=logfile)
     return observer
 
 
-def get_text_file_observer(name="nucypher.log", path=USER_LOG_DIR):
+def get_text_file_observer(name=DEFAULT_LOG_FILENAME, path=USER_LOG_DIR):
     _ensure_dir_exists(path)
     logfile = LogFile(name=name, directory=path, rotateLength=MAXIMUM_LOG_SIZE, maxRotatedFiles=MAX_LOG_FILES)
     observer = FileLogObserver(formatEvent=formatEventAsClassicLogText, outFile=logfile)
     return observer
+
+
+class Logger(TwistedLogger):
+    """Drop-in replacement of Twisted's Logger, patching the emit() method to tolerate inputs with curly braces,
+    i.e., not compliant with PEP 3101.
+
+    See Issue #724 and, particularly, https://github.com/nucypher/nucypher/issues/724#issuecomment-600190455"""
+
+    CURLY_BRACES_REGEX = re.compile('{+|}+')
+
+    @classmethod
+    def escape_format_string(cls, string):
```

Comment:
Does it really have to be a classmethod? Since it doesn't actually create an instance of `cls`, perhaps a standalone function would do.